lib/arbitrary.js

```diff
@@ -150,6 +150,67 @@ function oneof() {
   });
 }
 
+// Return a lazy arbitrary that delegates to another arbitrary at its
+// 'strict' property. An arbitrary must be assigned to that property before
+// this arbitrary can generate anything.
+function lazyArbitrary() {
+  var arb = {};
+  // This function must be pure because it will not be called with
+  // meaningful context.
+  arb.generator = generator.bless(function (size) {
+    return arb.strict.generator(size);
+  });
+  arb.shrink = shrink.noop;
+  arb.show = show.def;
+  arb = arbitraryBless(arb);
```

Comment:
the `arbitraryBless` could rebless the fields too. No need to do this, just a mental not for myself.